playwright.config.ts

```diff
@@ -27,7 +27,7 @@ const config: PlaywrightTestConfig = {
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: 0,
+  retries: process.env.CI ? 2 : 0,
```

Comment:
Do you have trouble with rebasing?
This change is already in `main`: https://github.com/redhat-developer/consoledot-e2e/blob/main/playwright.config.ts#L30

Reply:
No, but that change must be carried out when rebasing and there is no conflict